build/NuSpecs/ProjectReunion-Nuget-Common.targets

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Check TargetPlatformMinVersion during build to block unsupported configurations. -->
+  <Target Name="MicrosoftProjectReunionCheckTargetPlatformVersion" BeforeTargets="PrepareForBuild" 
+      Condition="'$(TargetPlatformVersion)' != '' and '$(TargetPlatformMinVersion)' != '' and '$(SkipMicrosoftProjectReunionCheckTargetPlatformVersion)'==''">
+    <PropertyGroup>
+      <MicrosoftProjectReunionTargetPlatformMinCheckValue>$([System.Version]::Parse('$(TargetPlatformMinVersion)').Build)</MicrosoftProjectReunionTargetPlatformMinCheckValue>
+    </PropertyGroup>
+    <Error 
+        Text="Microsoft.UI.Xaml nuget package requires TargetPlatformMinVersion &gt;= 10.0.15063.0 (current project is $(MicrosoftProjectReunionTargetPlatformMinCheckValue))" 
+        Condition="$(MicrosoftProjectReunionTargetPlatformMinCheckValue) &lt; 15063" />
```

Comment:
Update these error messages

Reply:
thanks, good catch.
I specified 17763 (== RS5) for min version.  This has been our target for the November Project Reunion "preview". 
Max version is 18362, this is 19H1.  Did we want this to be 18363? (19H2)?  Or we are specifically excluding 19H2? (Windows 10 Version 1909) per a handy table at: https://support.lenovo.com/us/en/solutions/ht502786 

And I assume xaml has some process whereby they will update this to 19041 for the 19041 May 2020 build?

Reply:
Wikipedia's got a similar handy table https://en.wikipedia.org/wiki/Windows_10_version_history

Reply:
per email, left as-is, we will resolve this (move to a newer max tested version) in a later PR.